### PR TITLE
21034 ep monitor classreset should not create a new instance if current is already empty

### PR DESCRIPTION
**Pull request by VincentBlondeau**
https://pharo.fogbugz.com/f/cases/21034/EPMonitor-class-reset-should-not-create-a-new-instance-if-current-is-already-empty

### CODE REVIEW

**Review by jecisc — APPROVED** [review]

**Review by tinchodias — CHANGES_REQUESTED** [review]

File: src/Epicea/EpRenameClassRefactoring.class.st [hunk 1]
```diff
@@ -21,10 +21,9 @@ EpRenameClassRefactoring class >> rename: oldName to: newName [
 
 { #category : #converting }
 EpRenameClassRefactoring >> asRBRefactoring [
-
-	^ RBRenameClassRefactoring new
```
[the remark this answers is not in the document]
Comment (VincentBlondeau) [reply src/Epicea/EpRenameClassRefactoring.class.st]: No it was not. I thought it was an automatic refactoring that was let uncommitted

[the remark this answers is not in the document]
Comment (tinchodias) [reply src/Epicea/EpRenameClassRefactoring.class.st]: So what do we proceed? could you remove this change and commit to this branch?

**Comment by VincentBlondeau** [comment]
@tinchodias: Ok for me

**Review by tinchodias — APPROVED** [review]
ok for me

**Comment by estebanlm** [comment]
but I hate your formatting ;)

**Pull request merged**